simapp/ante.go

```diff
@@ -31,6 +31,8 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		return nil, errors.New("sign mode handler is required for ante builder")
 	}
 
+	// TODO: remove this when ci is green, this is used to force ci to run
+
```

Comment:
Kinda confused on what "this" is. Maybe it's just a change in any Go file will force CI to run. If so we can merge and then delete this. Or alternatively modify CI to always run instead of skipping when no Go files are touched b/c that optimization seems like low value add.

```suggestion
```

Reply:
It was needed for sdk as ci became a bottleneck as free runners were all consumed. I can remove here